### PR TITLE
Add Everlaw to Other category

### PR DESCRIPTION
**Pull request by aflevin**

### CODE REVIEW

File: _data/other.yml [hunk 1]
```diff
@@ -82,6 +82,13 @@ websites:
       twitter: envato
       tfa: No
 
+   - name: Everlaw
```
Comment (Carlgo11) [inline _data/other.yml]: You missed one space in front of `- name: Everlaw`

**Comment by aflevin** [comment]
Sorry about that, and thanks for letting me know! Hope this fixes it.

**Comment by jamcat22** [comment]
@aflevin Looks like you fixed it! Another contributor will review it before it's merged. After that, it can take some time before it goes live on the site. Check out our [wiki](https://github.com/2factorauth/twofactorauth/wiki/FAQ_Merge) for more info.

**Comment by aflevin** [comment]
@jamcat22 Thank you!!

**Comment by fpigerre** [comment]
Thanks for the pull request @aflevin! :+1:

**Pull request merged**